### PR TITLE
Some needed changes

### PR DESCRIPTION
**Pull request by Jimobob**
Changes included in this PR:
- Evaluates arithmetic expressions that contain three OR more values. Evaluation is done from left -> right
- Added some execution statements for printing purposes (Remove the extra test helper methods for final)
- Division by zero error message
- Used setter methods for clarity

### CODE REVIEW

**Review by a1re1 — APPROVED** [review]
👍 nice. This was working for me.  I went ahead and merged these changes into this branch: https://github.com/rmh2805/Jott_Interpreter/tree/cleanup so that we're ready to merge everything into master whenever.

All of his samples work, but I'll do some more testing to see if there are any other problems I can find.

File: src/parseTree/nodes/str_literal.java [hunk 1]
```diff
@@ -27,15 +27,10 @@ public void fixChildren() {
         //todo Assign the proper children to their fields
         if (children.get(0) instanceof str_token) {
             data = ((str_token) children.get(0));
-            preQuote = new quote(data.getLineNumber());
-            endQuote = new quote(data.getLineNumber());
-            alias = null;
+            this.str_literal_set(new quote(data.getLineNumber()), data, new quote(data.getLineNumber()), null);
```
Comment (antxln) [inline src/parseTree/nodes/str_literal.java:30]: nit: We dont need quotes anymore

File: src/parseTree/nodes/str_literal.java [hunk 2]
```diff
@@ -46,15 +41,16 @@ public void fixChildren() {
     public str_literal() {
     }
 
-    public str_literal(quote preQuote, str_token data, quote endQuote) {
-        if (preQuote == null || data == null || endQuote == null) {
+    public void str_literal_set(quote preQuote, str_token data, quote endQuote, id alias) {
+        if (preQuote == null && data == null && endQuote == null && alias == null) {
```
Comment (antxln) [inline src/parseTree/nodes/str_literal.java:45]: Without quotes, this conditional may be `if (data == null && alias == null) {` which IMO makes more sense. I see how it works though

**Review by antxln — COMMENTED** [review]
Good work

File: src/parseTree/nodes/str_literal.java (hunk 2 above)
Comment (rmh2805) [reply src/parseTree/nodes/str_literal.java:45]: That was just a bodge to get it to fix children

**Review by rmh2805 — APPROVED** [review]

**Pull request merged**